src/zope/app/apidoc/static.py

```diff
@@ -159,10 +159,23 @@ def setUserAndPassword(self, user, pw):
         self.addHeader('Authorization', 'Basic %s:%s' % (user, pw))
 
     @classmethod
-    def begin(cls):
-        # TODO: We need to let this define what config file to execute.
+    def begin(cls, package_name='', zcml_file='configure.zcml'):
```

Comment:
I think `begin(cls, package_name=None, zcml_file=None)` is better here. No need to repeat the default value that's in the command line argument parser.